docs/awscloud.rst

```diff
@@ -120,26 +167,25 @@ If you don't use Conda and install the AWS CLI using something like `pip <https:
 command will attempt to run using the version of python found in the running container which won't be able to find
 the necessary dependencies.
 
-The following snippet shows how to install AWS CLI with `Miniconda <https://conda.io/miniconda.html>`_::
+The following snippet shows how to install AWS CLI with `Miniconda <https://conda.io/miniconda.html>`_ in the home folder::
 
+    cd $HOME
     sudo yum install -y bzip2 wget
     wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
     bash Miniconda3-latest-Linux-x86_64.sh -b -f -p $HOME/miniconda
     $HOME/miniconda/bin/conda install -c conda-forge -y awscli
     rm Miniconda3-latest-Linux-x86_64.sh
 
-When complete verifies that the AWS CLI package works correctly::
+When complete, verify that the AWS CLI package works correctly::
 
     $ ./miniconda/bin/aws --version
-    aws-cli/1.11.120 Python/3.6.3 Linux/4.9.43-17.39.amzn1.x86_64 botocore/1.5.83
```

Comment:
I think that it might be better if we move to `v2` of the `aws` CLI?

Reply:
@abhi18av That's a good point.

I tried to install awscliv2, but there are 2 consequences:

1) this version of the CLI seems to run inside a docker container:
<pre>
sh-4.2$ ./miniconda/bin/awsv2 --version
1.5.0
AWS CLI v2 command: docker run -i --rm -v /home/ec2-user/.aws:/root/.aws -v /home/ec2-user:/aws amazon/aws-cli
aws-cli/2.2.7 Python/3.8.8 Linux/4.14.231-173.361.amzn2.x86_64 docker/x86_64.amzn.2 prompt/off
</pre>
Would that work when called within the task's container?

2) the command name has changed, it's not `aws`, but `awsv2`. So, a switch to this new version (provided it is backward compatible) would require changes in nextflow or suggesting the user to create a symlink named `aws` to `awsv2`.

More in general, I believe that deciding whether to move to CLI2 (and probably also to ASW SDK 2, quite overdue IMHO) goes beyond the scope of this PR. Here, I was merely focusing on the documentation.

Reply:
thanks manule, the aws cli does not work on alpine (without some hacks), fair to stay with miniconda installation I would say.

Reply:
> thanks manule, the aws cli does not work on alpine (without some hacks), fair to stay with miniconda installation I would say.

I agree.

Reply:
> thanks manule, the aws cli does not work on alpine (without some hacks), fair to stay with miniconda installation I would say.

btw, I forgot to mention that my installation of cli 2 was also done with miniconda. I knew about the alpine issues.

Reply:
I would say to stay on v1 then

Reply:
> I would say to stay on v1 then

Agree, again :)